backend/src/EntityExtractor.Web/Program.cs

```diff
@@ -1,7 +1,20 @@
+using eKultura.EntityExtractor.Contracts;
+using eKultura.EntityExtractor.Domain;
+using System.IO.Abstractions;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
+builder.Services.AddSingleton<IFileSystem, FileSystem>();
+builder.Services.AddScoped<IFileStoring>(provider =>
+{
+    var fileSystem = provider.GetRequiredService<IFileSystem>();
+    var baseFolder = Path.Combine(Directory.GetCurrentDirectory(), "StoredFiles");
```

Comment:
This is not configurable, though. We want to have the target folder provided via appsettings.json